### ISSUE TITLE
Defect Hunting: Notification bugs

### ISSUE THREAD

**Issue by mikerodonnell89** [SAP/fundamental-ngx#7821]
From @Veerakarthick in this comment: https://github.com/SAP/fundamental-ngx/pull/7330#issuecomment-1048430725

- [ ] Observed Behavior:
The item count for the items available within the More menu is not read on navigating within the items.
The speech output is:
Approve
To move through items press up or down arrow.
Reject
Forward

Expected Behavior:
Ideally, the item count for the items available within the menu should be read on navigation.

- [ ] Observed Behavior:
When resizing the screen to ‘Responsive Design’ (320) pixels, the texts within the card are truncate no alternative provided.
Expected Behavior:
Ideally, the text must be adapted to the responsive design and should be within the card without loss of content.

- [ ] Observed Behavior:
On accessing the ‘Close’ button, the message strip information is not read for the user.
Expected Behavior:
Ideally, all important information visually conveyed to the user must be read

- [ ] notification core - [example](https://fundamental-ngx.netlify.app/#/core/notification) - click on the buttons `Open Notification` from both examples (Template as Content and Component as Content) and you will see both notifications are overlapping
<img width="690" alt="Screen Shot 2022-02-14 at 11 27 02" src="https://user-images.githubusercontent.com/4380815/153904597-66054940-965f-4282-8a12-b71a23696774.png">
